src/components/standalone/openvpn_tunnel/TunnelTable.vue

```diff
@@ -186,6 +190,27 @@ function getCellClasses(item: ServerTunnel | ClientTunnel) {
         </p>
       </div>
     </template>
+    <template #state="{ item }: { item: ServerTunnel | ClientTunnel }">
```

Comment:
```suggestion
    <template #connected="{ item }: { item: ServerTunnel | ClientTunnel }">
```
The issue with the connected field not showing is probably simply because of this :P

Reply:
Nailed it!